blocks/comparison/comparison.js

```diff
@@ -0,0 +1,41 @@
+import { getLibs } from '../../scripts/utils.js';
+
+/**
+ * @param {Element} el
+ */
+const init = async (el) => {
+  const { createTag } = await import(`${getLibs()}/utils/utils.js`);
+
+  const rows = Array.from(el?.querySelectorAll(':scope > div'));
+  const headers = Array.from(rows.shift().children);
+
+  const headersRow = createTag('tr');
+  headers.forEach((cell) => {
+    headersRow.appendChild(createTag('th', { scope: 'col' }, cell));
+  });
+  const thead = createTag('thead', {}, headersRow);
+
+  const tbody = createTag('tbody');
+  rows?.forEach((row, idx) => {
```

Comment:
`rows` will never be undefined/null. `querySelectorAll` returns an empty nodelist if there aren't any matches
```suggestion
  rows.forEach((row, idx) => {
```

Reply:
Thanks